<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart House</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="led1Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>131</width>
      <height>110</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Turn ON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>ĐÈN 1</string>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/ledOff.png</normaloff>
      <normalon>:/images/ledOn.png</normalon>:/images/ledOff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="led2Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Turn ON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>ĐÈN 2</string>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/ledOff.png</normaloff>
      <normalon>:/images/ledOn.png</normalon>:/images/ledOff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="led3Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Turn ON&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>ĐÈN 3</string>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/ledOff.png</normaloff>
      <normalon>:/images/ledOn.png</normalon>:/images/ledOff.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>ĐÈN 1 TẮT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>ĐÈN 2 TẮT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>ĐÈN 3 TẮT</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trạng thái</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>326</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chế độ công tắc</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="switchMode1Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/disable.png</normaloff>
      <normalon>:/images/enable.png</normalon>:/images/disable.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="switchMode2Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/disable.png</normaloff>
      <normalon>:/images/enable.png</normalon>:/images/disable.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="switchMode3Button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/disable.png</normaloff>
      <normalon>:/images/enable.png</normalon>:/images/disable.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="autoMode1Button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>89</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/manual.png</normaloff>
      <normalon>:/images/auto.png</normalon>:/images/manual.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="pir1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="smart_house.qrc">
      <normaloff>:/images/grey.png</normaloff>
      <normalon>:/images/red.png</normalon>:/images/grey.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HC-SR501</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="smart_house.qrc"/>
 </resources>
 <connections/>
</ui>
